Library/Formula/camlp5.rb

```diff
@@ -13,20 +13,33 @@ class Camlp5 < Formula
 
   deprecated_option "strict" => "with-strict"
   option "with-strict", "Compile in strict mode"
+  option "with-doc", "Install the html and info documentation"
+  option "with-latex", "Install the pdf, ps, and tex documentation"
```

Comment:
Please try to make this just `with-tex` and then `depends_on :tex => :optional`. The `if build.with?` dependencies are pretty horribly handled.